.github/workflows/package.yml

```diff
@@ -403,3 +403,54 @@ jobs:
           allowUpdates: true
           updateOnlyUnreleased: true
           artifacts: "${{ env.PACKAGE_DIR }}.tar.gz"
+
+  build-deb:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      attestations: write
+      id-token: write
+    env:
+      BUILD_DIR: Dist
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install -y --no-install-recommends cmake make g++ libpcap-dev
```

Comment:
This step fails because we don't run it as `sudo`, which made me notice that this runs directly on the GitHub runner VM. Why not run it inside one of our containers like `seladb/ubuntu2404`? This image should already have these dependencies pre-installed